packages/@aws-cdk/core/README.md

```diff
@@ -17,6 +17,42 @@ Guide](https://docs.aws.amazon.com/cdk/latest/guide/home.html) for
 information of most of the capabilities of this library. The rest of this
 README will only cover topics not already covered in the Developer Guide.
 
+## Stacks and Stages
```

Comment:
> Started to write a section in the README on Stacks and
Stages, but I couldn't figure out how to write about the
use for Stages without referencing unreleased things, so
I gave up.

I think it's ok to mark sections as future dated, or not supported yet.

On the other hand, you could put it in another markdown that sits alongside that can then get merged into the README when ready. It could be called `stage-design.md` for example.

The mechanism we have on our team for this is an RFC, similar to the feature flags one. Maybe a stripped down version?

Reply:
I'd like to learn about Stages to effectively review this PR. If it's written elsewhere, can you point me?

Reply:
I am okay with adding a note that this is not yet officially supported.